content/engineering/languages-runtimes/ruby.md

```diff
@@ -10,34 +10,27 @@ eleventyNavigation:
   key: Ruby
   order: 5
   title: Ruby
+subnav:
+  - text: Style guide
+    href: "#style-guide"
+  - text: Testing
+    href: "#testing"
 ---
-Help us make this section better by
-[submitting an issue](https://github.com/18F/development-guide) or joining us
-in the [#ruby](https://18f.slack.com/messages/ruby/) channel!
+Help us make this section better by [submitting an issue](https://github.com/18F/development-guide) or joining us in the [#ruby](https://18f.slack.com/messages/ruby/) channel!
 
 A guide for writing and maintaining Ruby and Rails applications
 
-## Style Guide
+## Style guide
+Follow the [Ruby style guide](https://github.com/bbatsov/ruby-style-guide) and enforce it via static analysis tools such as [Code Climate] and [Rubocop]. You can copy the [recommended Rubocop configuration] ({{ "/assets/engineering/dist/rubocop.yml" | url }}) in your Ruby project and make any changes based on your team's preferences.
```

Comment:
Poorly formatted link for Rubocop configuration (extra space after the "]" is likely breaking it)

Reply:
Fixed in d8717cd